src/main/java/storage/WriteDataFile.java

```diff
@@ -0,0 +1,105 @@
+package storage;
+
+import cheatsheet.CheatSheet;
+
+import java.io.IOException;
+import java.io.FileWriter;
+
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import static storage.ParseCheatSheetFile.NAME;
+import static storage.ParseCheatSheetFile.PROGRAMMING_LANGUAGE;
+import static storage.ParseCheatSheetFile.DETAILS;
+
+/**
```

Comment:
perhaps remove  some of the old Duke descriptions

Reply:
hmm isn't that his own functions and not the old Duke functions?

Reply:
er like the line i commented on said duke.txt and list of tasks